os/include/tinyara/pm/pm.h

```diff
@@ -482,20 +482,18 @@ int pm_timedsuspend(int domain_id, unsigned int milliseconds);
  * Name: pm_suspendcount
  *
  * Description:
- *   This function is called to get current stay count.
+ *   This function is called to get current suspend count of domain.
  *
  * Input Parameters:
  *   domain_id - The domain ID of the PM activity
  *
  * Returned Value:
- *   Current pm stay count
- *
- * Assumptions:
- *   This function may be called from an interrupt handler.
+ *   non-negative interger: the suspend count of domain
```

Comment:
Pls use capital letter for first character

Reply:
Done